gcp/service_auth/tests/test_get_token.py

```diff
@@ -7,33 +8,47 @@
 import logging
 import datetime
 from pathlib import Path
+import json
 
-from gcp_service_auth import GcpIdentityToken
+try:
+    from gcp_service_auth import GcpIdentityToken
+except ModuleNotFoundError:
+    sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+    from gcp_service_auth import GcpIdentityToken
+    pass
+                                 
 
-TEST_URL = os.environ.get('TEST_URL')
 
 @pytest.fixture(scope="module")
 #@pytest.mark.with_op
-def gcp_browse_cred() -> Generator[str, str, str]:
+def gcp_browse_cred() -> (Generator[str, str, str], str):
```

Comment:
I'm confused.  You yield (cred_file, test_url), then return "". Neither matches the signature.  

You probably want the return type to be Generator[str, str] and not have the final return "" .

Reply:
TRUE. My brain was working for yield.